include/bm/bluetooth/services/ble_nus.h

```diff
@@ -67,13 +67,25 @@ void ble_nus_on_ble_evt(ble_evt_t const *ble_evt, void *context);
 
 /** @brief Nordic UART Service event types. */
 enum ble_nus_evt_type {
-	/** Data received. */
+	/**
+	 * @brief Error event.
+	 */
+	BLE_NUS_EVT_ERROR,
```

Comment:
nit: I would keep the `_ERROR` event last in the enum everywhere.

Reply:
It is the first everywhere so that it has the same value for all libraries and services. Having it at the end will give different values, and it will change if another event is added.